lifecycle_msgs/msg/TransitionDescription.msg

```diff
@@ -1,3 +1,8 @@
+# The transition id and label of this description
```

Comment:
punctuation

Reply:
Addressed in https://github.com/ros2/rcl_interfaces/pull/91